services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -455,8 +456,12 @@ async def get_service_pricing_plan(
             cookies=self.session_cookies,
         )
         response.raise_for_status()
-        data = Envelope[ServicePricingPlanGet].parse_raw(response.text).data
-        return data
+        pricing_plan_get = Envelope[PricingPlanGet].parse_raw(response.text).data
+        if pricing_plan_get:
+            return parse_obj_as(
+                ServicePricingPlanGet, pricing_plan_get.dict(exclude={"is_active"})
+            )
```

Comment:
Perhaps you could keep `data = Envelope[ServicePricingPlanGet].parse_raw(response.text).data` and then where `ServicePricingPlanGet` you could add an assert that it has exactly the same fields as `ServicePricingPlanGet` except for the `is_active_fields`. That way if these two models get out of sync even importing that module would fail.